plugins/generator-1.15.2/forge-1.15.2/utils/boundingboxes.java.ftl

```diff
@@ -57,10 +54,10 @@
         makeCuboidShape(${box.mz}, ${box.my}, ${16 - box.mx}, ${box.Mz}, ${box.My}, ${16 - box.Mx})
     <#elseif facing == "up">

```

Comment:
Old code for calling this method was

```
case EAST:
            case WEST:
                <@makeBoundingBox positiveBoxes negativeBoxes noOffset "up"/>
```

but now "up" is not used with log rotation. Is this correct?

Reply:
Correct, but it's still being used by other rotation types

Reply:
It is not that I was wondering why it is not deleted, I am wondering how this works now as I would expect

`<#elseif facing == "up">`

to be something like

`<#elseif facing == "up" || facing == "SOME_AXIS,X,YORZNOTSURE">`

Reply:
Y is the default state, so it just uses the default bounding box (the final else), X is its own case and the Z bounding box is the same as `down`